### PR TITLE
answer:12

### PR DESCRIPTION
**Pull request by mikelgo**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

**Comment by tomalaforge** [comment]
Thanks for submitting your solution, nice work 👍

File: apps/scroll-cd/src/app/app.component.ts [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
+import { BehaviorSubject, distinctUntilChanged, fromEvent, tap } from 'rxjs';
+import { NgxDirtyCheckerModule } from '@code-workers.io/ngx-dirty-checker';
 
 @Component({
   standalone: true,
-  imports: [NgIf, AsyncPipe],
+  imports: [NgIf, AsyncPipe, NgxDirtyCheckerModule],
   selector: 'app-root',
   template: `
+    <ngx-dirty-checker></ngx-dirty-checker>
```
Comment (tomalaforge) [inline apps/scroll-cd/src/app/app.component.ts:17]: What is this for ?

Comment (mikelgo) [reply apps/scroll-cd/src/app/app.component.ts:17]: That's just a helper component to visualize the number of dirty checks/ re-renderings of a component. 

https://github.com/code-workers-io/ngx-dirty-checker

However I forgot to commit the package

Comment (tomalaforge) [reply apps/scroll-cd/src/app/app.component.ts:17]: Thanks 👍

Didn't know about that library. Can be useful in dev mode.

**Pull request closed without merging**